### PR TITLE
[DC-1590] Reduce table creation and drop for ID deduplicate test

### PR DESCRIPTION
**Pull request by nishanthpp93**
* [DC-1590] Reduce table creation-drop instances to just once

[DC-1590]: https://precisionmedicineinitiative.atlassian.net/browse/DC-1590

### CODE REVIEW

**Comment by amcgrenera-vumc** [comment]
This looks good to me and is passing locally, only question, how come you added the additional fields? And should those be included in the `fields` and `cleaned_values` in `tables_and_counts`?

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/id_de_duplicate_test.py [hunk 1]
```diff
@@ -39,11 +39,11 @@ def setUpClass(cls):
                                                 cls.sandbox_id)
 
         # Generates list of fully qualified table names
-        for table_name in [CONDITION_OCCURRENCE, OBSERVATION]:
+        for table_name in cls.affected_tables:
             cls.fq_table_names.append(
```
Comment (amcgrenera-vumc) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/id_de_duplicate_test.py:43]: Good call on using this instead of explicitly listing!

**Comment by nishanthpp93** [comment]
> This looks good to me and is passing locally, should the additional required fields you added be included in the `fields` and `cleaned_values` in `tables_and_counts`?

That's a good point. I know the cleaning rule should not affect them, but it is worth checking and making sure that they are unaffected.

**Review by amcgrenera-vumc — APPROVED** [review]
thanks for adding that in! this looks good to me

**Pull request merged**